sdk/template/azure-sdk-template/swagger/README.md

```diff
@@ -0,0 +1,149 @@
+# Azure Template for Java
+
+> see https://aka.ms/autorest
+
+This is the template AutoRest configuration file for client SDKs.
+---
+## Getting Started
+
+To build the SDK, simply [Install AutoRest](https://aka.ms/autorest) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+### Setup
+
+Fork and clone [autorest.java](https://github.com/Azure/autorest.java) and run the following:
+
+```ps
+git checkout main
+git submodule update --init --recursive
+mvn install -Dlocal -DskipTests
+npm install
+npm install -g autorest
+```
```

Comment:
We should not use AutoRest "from source" builds to generate code as it can be unstable and may have features that are not yet ready to be released.

Reply:
Given that this PR runs many pipelines I'll submit a follow-up PR